Sources/SwiftLayout/Core/Anchors/Anchors.swift

```diff
@@ -110,6 +110,36 @@ public enum Anchors {
         return container
     }
     
+    /// combination of **top, leading, bottom** anchors for superview
+    ///
+    /// - Parameter offset: plus constant at top, plus constant at leading, minus constant at bottom. default value is 0.
+    /// - Returns:
+    /// superview.top + constant(offset)
+    /// **AND**
+    /// superview.leading + constant(offset) and superview.bottom - constant(offset)
+    public static func startBracket(offset: CGFloat = .zero) -> AnchorsContainer {
```

Comment:
I'll think about it too, but is there a better name?

Reply:
@gmlwhdtjd @della-padula 

i think, also, good timing for discuss to cap and shoe.

an idea for merge function instead currents.

```swift
sides(.leading, .top, .trailing, offset: 12.0)
```

Reply:
Oh, It's good idea

Reply:
It would be good to receive a list of directions as a parameter and modify it to apply constraints while it is ethered.

Reply:
@oozoofrog @della-padula 

I have a question.
If the sides method can receive multiple attributes using array or Variadic parameters
How should I handle the rest of the elements except leading, trailing, top, bottom ?

It sounds like we need a way to explicitly restrict the user's input.

Reply:
> @oozoofrog @della-padula
> 
> I have a question. If the sides method can receive multiple attributes using array or Variadic parameters How should I handle the rest of the elements except leading, trailing, top, bottom ?
> 
> It sounds like we need a way to explicitly restrict the user's input.

how about this?

```swift
side(x: ..., y: ...)
```

Reply:
> 

How about defining a enum that contains only four items, and then converting the type from the internal logic?
`enum Axis {
    case leading
    case trailing
    case top
    case bottom

    var convertedType: OURTYPE {
        ....
    }
}`